subsys/bluetooth/host/direction.c

```diff
@@ -40,7 +40,7 @@ static struct bt_le_df_ant_info df_ant_info;
 const static uint8_t df_dummy_switch_pattern[BT_HCI_LE_SWITCH_PATTERN_LEN_MIN] = { 0, 0 };
 #endif /* CONFIG_BT_DF_CONNECTIONLESS_CTE_RX */
 
-#define DF_SUPP_TEST(feat, n)                   ((feat) & BIT((n)))
+#define DF_SUPP_TEST(feat, n)                   ((feat) & (n))
 
 #define DF_AOD_TX_1US_SUPPORT(supp)             (DF_SUPP_TEST(supp, \
 						BT_HCI_LE_1US_AOD_TX))
```

Comment:
Change all `DF_XXX_SUPPORT` macros this way:
```suggestion
#define DF_AOD_TX_1US_SUPPORT(supp)             ((supp) & BT_HCI_LE_1US_AOD_TX)
```

Reply:
Thanks for your suggest, yes! you are right. I modified the changes as you have suggested.